packages/react-components/react-motions-preview/src/factories/createMotionComponent.ts

```diff
@@ -23,7 +21,7 @@ type MotionComponentProps = {
  */
 export function createMotionComponent(motion: AtomMotion | AtomMotionFn) {
   const Atom: React.FC<MotionComponentProps> = props => {
-    const { children, iterations = 1, imperativeRef } = props;
```

Comment:
I'm fine with this simplification; I don't think the use case of adding iterations is so common that not having a custom prop for it would be onerous.